src/vehicle.h

```diff
@@ -504,7 +504,7 @@ class vehicle : public JsonSerializer, public JsonDeserializer
     nc_color part_color( int p, bool exact = false ) const;
 
     // Vehicle parts description
-    int print_part_desc (WINDOW *win, int y1, int width, int p, int hl = -1) const;
+    int print_part_desc (WINDOW *win, int y1, const int max_y, int width, int p, int hl = -1) const;
```

Comment:
Just a style note: a top level `const` of a parameter in a function *declaration* is effectively ignored. The declarations `void foo(const int);` and `void foo(int);` are the same. It does not matter to the caller at all, therefor it's usually better to omit the useless `const`.

The top level `const` only has a meaning in the definition, as in `void foo(const int) { ... }`, and only inside the function body.

And to avoid any misconception: the `const` in `void func(const int &p);` is *not* a top level `const`, it applies to `int`, not to `p` and is therefor not ignored.